vrplib/parse/parse_vrplib.py

```diff
@@ -79,6 +77,9 @@ def group_specifications_and_sections(lines: list[str]):
             end_section = start + 1
 
             for next_line in lines[start + 1 :]:
+                if ":" in next_line:
```

Comment:
Is this a sufficient test for a specification? (probably yes but can there ever be data in a section with a colon in it?)

Reply:
I cannot think of any data section that must include `:`'s. Only thing I can think of are strings but we should probably avoid strings in data sections as they can often be mapped to numerical values. I think we're OK if we define the VRPLIB standard to reserve `:` as a specifications identifier.

Reply:
I am excited about having a grammar for this so we can definitively say what's OK and what's not 😆.